awspub/sns.py

```diff
@@ -23,13 +24,13 @@ class SNSNotification(object):
     structuring rules for SNS notification JSON
     """
 
-    def __init__(self, context: Context, image_name: str, region_name: str):
+    def __init__(self, context: Context, image_name: str, s3_bucket_region: str):
```

Comment:
you can get the bucket region from the context. no need to add this as a parameter here.

Reply:
I imported S3 class and get the `bucket_region` directly from it with context, instead of passing it as a separate parameter.